FILES.deploy-tools/root/playbooks/ndslabs-k8.yml

```diff
@@ -66,21 +77,44 @@
 # Now to kubernetes, back again after
 ##########################################################################################################################
 - include: /usr/local/lib/kubernetes/contrib/ansible/cluster.yml
+ 
+##########################################################################################################################
+# Clean up known K8's troubles
+##########################################################################################################################
+
+- name: stop and disable kube-addons
+  hosts: masters
+  become: true
+  tasks:
+    - service: name=kube-addons state=stopped enabled=false
+
 
 ##########################################################################################################################
 # Now additional setup now that the cluster is running
 ##########################################################################################################################
 
+- name: LoadBalancer public interface
+  hosts: loadbal
+  gather_facts: false
+  connection: local
+  tasks:
+    - os_floating_ip: server="{{ inventory_hostname }}" 
```

Comment:
I think this should be server="{{ logical_cluster_name }}-{{ inventory_hostname }}"